### PR TITLE
Add bundle config

### PR DESCRIPTION
**Pull request by huwshimi**
Display the config for the services in a bundle. As noted in #1514.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- load the details for a bundle
- below the config should be a list of services and they should expand to display the config options

File: jujugui/static/gui/src/app/components/entity-content/entity-content.js [hunk 1]
```diff
@@ -32,37 +32,115 @@ YUI.add('entity-content', function() {
...
+        };
+        if (optionsList.length === 0) {
+          optionsList.push(
+            <div key="none">
+              No config options for this service.
+            </div>);
+        }
+        return (
+          <juju.components.ExpandingRow
+            classes={classes}
```
Comment (anthonydillon) [inline jujugui/static/gui/src/app/components/entity-content/entity-content.js]: Could you not just simply add `'entity-content__bundle-config'` here?

**Comment by anthonydillon** [comment]
Code and QA is good. But unfortunately design has an issue with using the new expanding card design for this section. They would like the bundle config to match the design and interaction of the [storefront's bundle configs](https://jujucharms.com/openstack-base/bundle/42#charms). But dont mind keeping it on the grey background to match the bundle details in the GUI.

Design justifications included that it feels to grand for supplementary content. It also scrolls under the sticky header which brakes the layer logic.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5138/
Test PASSed.

File: jujugui/static/gui/src/app/components/entity-content/entity-content.js (hunk 1 above)
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/entity-content/entity-content.js]: Unfortunately this is how we have to do it with the way the classNames util expects the object.

**Comment by huwshimi** [comment]
@abentley I've updated the expanded styles.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5142/
Test PASSed.

**Comment by abentley** [comment]
@huwshimi Did you mean @anthonydillon ?

**Comment by anthonydillon** [comment]
@huwshimi looks great :+1:

**Comment by hatched** [comment]
Looks good! Thanks :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**